static-site/src/components/Groups/Tiles/index.tsx

```diff
@@ -21,20 +21,24 @@ export const GroupTiles = () => {
 };
 
 
-const createGroupTiles = (groups: Group[], colors = [...theme.titleColors]): GroupTile[] => groups.map((group) => {
-  const groupColor = colors[0]!;
-  colors.push(colors.shift()!);
-
-  const tile: GroupTile = {
-    img: "empty.png",
-    url: `/groups/${group.name}`,
-    name: group.name,
-    color: groupColor,
-    private: group.private
-  };
-
-  return tile;
-});
+function createGroupTiles(groups: Group[], colors = [...theme.titleColors]): GroupTile[] {
+  return groups
+    .sort((a, b) => a.name.localeCompare(b.name))
```

Comment:
Briefly wondered about using [numeric sorting](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/localeCompare#numeric_sorting), but probably not worth it since options arguments are not supported in all browsers yet.

Reply:
I also considered [faster alternatives](https://stackoverflow.com/q/14677060) but decided to keep as-is because:

1. `(a<b?-1:(a>b?1:0))` is case-sensitive which we don't want
2. The number of groups is small enough that I don't think the  `localeCompare` slowness matters much